engine/src/main/java/org/destinationsol/game/chunk/ChunkManager.java

```diff
@@ -38,8 +38,8 @@
     private final RemoveController backgroundRemoveController;
     private final ChunkFiller filler;
 
-    private int X;
-    private int Y;
+    private int x;
```

Comment:
I don't know this code very well, but why does the chunk manager have coordinates? o.O Maybe these internal members can be renamed to something more specific, which would free the names `x` and `y` as method parameters.

Reply:
The chunk manager appears to use the co-ordinates as the origin point from which it fills chunks, or at least that is what the source code indicates. I have re-named the variables to `originX` and `originY`.